Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -2066,6 +2132,131 @@ def get_item_as_eml(item_id, target_mailbox=None):     # pragma: no cover
         return file_result
 
 
+def collect_manual_attachments(manual_attach_obj):      # pragma: no cover
+    attachments = []
+    for attachment in manual_attach_obj:
+        res = demisto.getFilePath(os.path.basename(attachment['RealFileName']))
+
+        file_path = res["path"]
+        with open(file_path, 'rb') as f:
+            attachments.append(FileAttachment(content=f.read(), name=attachment['FileName']))
+
+    return attachments
+
+
+def process_attachments(attach_cids="", attach_ids="", attach_names="", manual_attach_obj=[]):      # pragma: no cover
```

Comment:
This is no good, as it makes the argument mutable.
```
manual_attach_obj=[]
```
Instead, init it to `None` and set to an empty list when its not populated - as you've done in the commented code below